examples/network/custom_region_demo.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2015, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import copy
+import csv
+import json
+import os
+import sys
+
+from pkg_resources import resource_filename
+
+from nupic.data.file_record_stream import FileRecordStream
+from nupic.encoders import MultiEncoder, ScalarEncoder, DateEncoder
+from nupic.engine import Network
+
+_VERBOSITY = 0  # how chatty the demo should be
+_SEED = 1956  # the random seed used throughout
+_INPUT_FILE_PATH = resource_filename(
+  "nupic.datafiles", "extra/hotgym/rec-center-hourly.csv"
+)
+_OUTPUT_PATH = "network-demo-output.csv"
+_NUM_RECORDS = 2000
+
+# Config field for IdentityRegion
+I_PARAMS = {
+    'dataWidth':1,
+}
+
+
+
+def createEncoder():
+  """Create the encoder instance for our test and return it."""
+  consumption_encoder = ScalarEncoder(21, 0.0, 100.0, n=50, name="consumption",
+      clipInput=True)
```

Comment:
Either move the first argument to a newline with this indentation or indent this to the same column as the first argument